### PR TITLE
[develop] Scenario Edit SLA expiration date 

### PR DESCRIPTION
**Pull request by samuelloza**

### CODE REVIEW

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountDetailsPage/LightningAccountDetailsPage.java [hunk 1]
```diff
@@ -22,19 +22,18 @@
...
+    private static final By SLA_EDITION_BUTTON = By.xpath("//button[@title='Edit SLA Expiration Date']");
 
     private void clickDetailsTab() {
-        GuiInteractioner.clickWebElement(detailsTab);
+        GuiInteractioner.clickWebElement(DETAIL_TAB);
     }
 
     private String getTextFromDetail(final String fieldName, final String tagType) {
-        return GuiInteractioner.getTextFromWebElement(By.xpath(String.format(accountInfoXpath, fieldName, tagType)));
+        return GuiInteractioner.getTextFromWebElement(By.xpath(String.format(ACCOUNT_INFO, fieldName, tagType)));
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountDetailsPage/LightningAccountDetailsPage.java:36]: Please, keep this By and Strings locators in lowerCase to be consistent to all the project, we use Upper case only for constants in primitive data type (int, Strings, char) but not applicable for locators.

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountDetailsPage/LightningAccountDetailsPage.java [hunk 2]
```diff
@@ -22,19 +22,18 @@
 public class LightningAccountDetailsPage extends BasePage implements IAccountDetailsPage {
 
     private static final int ACCOUNT_STRING_SIZE = 8;
-    private String accountInfoXpath = "//span[.='%1$s']/../../div[2]//%2$s";
-    private String opportunityToSearchXpath =
+    private static final String ACCOUNT_INFO = "//span[.='%1$s']/../../div[2]//%2$s";
+    private static final String OPPORTUNITY_TO_SEARCH =
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountDetailsPage/LightningAccountDetailsPage.java]: Please, keep this By and Strings locators in lowerCase to be consistent to all the project, we use Upper case only for constants in primitive data type (int, Strings, char) but not applicable for String locators, so avoid to put them as final.

File: salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/AccountSteps.java [hunk 3]
```diff
@@ -68,7 +69,7 @@ public void verifyAccountDataIsDisplayedAtDetails() {
         SoftAssert softAssert = new SoftAssert();
         actualAccountDetails.forEach((field, actualValue) -> {
             softAssert.assertTrue(actualValue.startsWith(expectedAccountDetails.get(field)),
-            String.format(incorrectAssertionMessage, field, "Account Details Page"));
+                    String.format(incorrectAssertionMessage, field, "Account Details Page"));
```
Comment (MirkoFer-622) [inline salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/AccountSteps.java]: Correct the identation please.

**Review by MirkoFer-622 — CHANGES_REQUESTED** [review]
Take account the few comments I left please and avoid to use Uppercase and final acces modifier in By objects and String locators, thanks.

**Comment by samuelloza** [comment]
Thanks for your comments @MirkoFer-622

**Review by MirkoFer-622 — APPROVED** [review]

**Pull request merged**